nginx/nginx.conf

```diff
@@ -0,0 +1,11 @@
+server {
+    listen 80;
```

Comment:
Is this in internal network? :D

Reply:
The NGINX will attach to port 80 (at least for now, since we will move the listening port to .env), which will further binds to another docker network to be accessed by the gateway container.

Reply:
Got it. To make this more safer, I think that there is an LB probably can be set above this network.